src/main/java/Sun.java

```diff
@@ -0,0 +1,93 @@
+import java.util.Scanner;
+public class Sun {
```

Comment:
```suggestion
import java.util.Scanner;

public class Sun {
```
I feel add a blank line between import and Class definition would make code easier to read and understand.